### ISSUE TITLE
ts-force generation in documentation

### ISSUE THREAD

**Issue by andymills** [ChuckJonas/bad-ass-salesforce-stack#64]
Hi, could you add as part of the setup process the instructions to generate the Salesforce Object classes please. 

npm run generate-ts-force

It wasn't obvious to me until a bit of digging, so it might be useful to add it.

Thanks
Andy

**Comment by ChuckJonas** [comment]
Have you seen this page?

https://ts-force.gitbook.io/ts-force/guides/code-generation

**Comment by andymills** [comment]
Yes thanks. Thats where I first saw the command. Then I looked in the package.json file and saw the script already set up and ready to go. I just had to go looking for it. If it was mentioned in the documentation/readme then I would have saved a bit of time.
